x-pack/legacy/plugins/siem/public/components/ml_popover/upgrade_contents.tsx

```diff
@@ -38,14 +46,24 @@ export const UpgradeContents = React.memo(() => {
         />
       </EuiText>
       <EuiSpacer />
-      <EuiButton
-        href="https://www.elastic.co/subscriptions"
-        iconType="popout"
-        iconSide="right"
-        target="_blank"
-      >
-        {i18n.UPGRADE_BUTTON}
-      </EuiButton>
+      <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none">
```

Comment:
Couple of changes to the props for `EuiFlexGroup`:

- Remove `justifyContent` prop.
- Make `gutterSize` prop `gutterSize="s"`.
- Add boolean prop `wrap`.